docs/installation.rst

```diff
@@ -119,19 +123,6 @@ Now we need to activate our new environment ::
 
   mamba activate openfe_env
 
-
-.. warning::
-
-   Installing on Macs with Apple Silicon requires a creating an x86_64
-   environment, as one of our requirements is not yet available for Apple
-   Silicon. Run the following modified commands
```

Comment:
I've been building openfe successfully on my apple silicon machine - what was the limiting dependency here?

Reply:
This is one of those where we just haven't updated the docs since Mike made it such that ambertools started supporting osx-arm64.